ospo-mindmap/Content/ospomindmap.md

```diff
@@ -70,34 +111,127 @@ development team of the Linux Project
+    - Publishing of project's release artifacts to public repositories such as Maven Central, npm, etc.
+  - Sponsoring Open Source
+    - Sponsoring third-party projects
+    - Sponsoring organization's projects or individual members
+    - Sponsoring events
+  - Signing contribution related agreements
+    - Contributor license agreement (individual / corporate)
+    - Developer certificate of origin
+  - Shutting down a new open source project
+- Using Open Source
+  - Using an open source project as a standalone tool
+  - Including open source snippets or files
+  - Including open source packages
+  - Forking an open source project
+  - Inventorying open source usage incl. packages, licensing and communities
+  - Maintaining a classification and enforcing rules for open source and proprietary licenses
+- Deciding on Open Source
+  - Drive Open Source Review Board with senior leaders from relevant stakeholder to make decisions on open source
+  - Drive Open Source Working Group with representatives / interested stakeholders throughout the organization with the aim of improving open source at the organization
+- Marketing and/or communicating Open Source
+  - Internal messaging
+    - Maintaining internal wiki outlining policy, processes, tooling and best practices
+  - External messaging
+    - Organization's open-source principles
+    - Simplified/redacted version of open source wiki
+    - Organization GitHub ReadMe
+    - Open Source landing page / portal
+    - Obligation fulfillment
+    - Organizing community events
+- Educating people
+  - Open source awareness which includes details on open source policy, processes, tools and the basics of open source licensing, community and contributions
+  - Guidelines and best practices for using of, contributing to, complying with open source
+  - Training on open source project management
+  - New employee orientation
+  - Checklist for product teams
+  - Checklist for developers
+  - Checklist for SW procurement
+  - Executing a mentorship for open source leaders and/or organization leaders
+  - Organizing events 
+    - Present Open Source at the organization's all-hands and similar meetings
+    - Organize brown bag seminars on open source topics (with invited speakers)
+  - Assist speakers for coding forums / conferences
+
+### 📈 Prioritize and Drive Open Source Upstream Development
+
+#### Create Educational Programs
 
 - Good OSS hygiene
 - Technical programs
   - Basics in OS tech stack
   - How to request features
   - How to file bug reports
   - How to contribute basic code
+  - How to do non-code contributions
+    - How to contribute code review
+    - How to contribute documentation
+    - How to contribute testing
+    - How to contribute to project management
+
+#### Create Internal Open Source Ambassadors
+
+- Drive event sponsorship
+- Train speakers for coding forums
 
-#### Optimize open outbound source contributions
+#### Contributes to Third-Party OSS Projects
```

Comment:
Some more contributions, these are for projects that have a user interface:
- Perform usability tests
- UX improvements
- Develop / improve graphics
- Perform localizations (translate to another language)

Reply:
Added 

```
- Perform tests such as hardware, usability, etc.
- Improve user experience
- Develop / improve graphics
- Create localizations (translate to another language)
```